src/qt/radiobox.cpp

```diff
@@ -168,10 +168,42 @@ static QAbstractButton *GetButtonAt( const QButtonGroup *group, unsigned int n )
 
 bool wxRadioBox::Enable(unsigned int n, bool enable)
 {
-    QAbstractButton *qtButton = GetButtonAt( m_qtButtonGroup, n );
-    CHECK_BUTTON( qtButton, false );
+    if( enable && !m_qtGroupBox->isEnabled() )
```

Comment:
```suggestion
    if ( enable && !m_qtGroupBox->isEnabled() )
```
There are some coding style guidelines that wxW uses, some of which may be unusual, but it's too late to adopt new ones so the best scenario is to be consistent and apply them in new code as well.

Reply:
Thanks for the feedback, will keep that in mind for future commits.